### PR TITLE
included new granular permissions

### PR DESCRIPTION
**Pull request by AndreasWolter**

### CODE REVIEW

**Comment by AndreasWolter** [comment]
#sign-off

**Comment by ShannonLeavitt** [comment]
@AndreasWolter - You've opened this PR in the public repo. Please point the changes to the private repo instead, as you have done [here](https://github.com/MicrosoftDocs/sql-docs-pr/pulls?q=is%3Apr+author%3AAndreasWolter).

Or if you must make these changes in the public repo, you can add a `#please-open` command and then ask William to sign off.

**Comment by WilliamDAssafMSFT** [comment]
@ShannonLeavitt, Andreas is now external to MSFT. Re-opening.

**Comment by WilliamDAssafMSFT** [comment]
Thanks @AndreasWolter for takin the time to contribute! Will review internally, and potentially if other articles need this information. If the article is updated with your commit, GitHub will list you as an article contributor.

Thanks again for this feedback. Your suggestions help improve our documentation.

**Comment by ShannonLeavitt** [comment]
#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/relational-databases/extended-events/quick-start-extended-events-in-sql-server.md [hunk 1]
```diff
@@ -49,7 +49,8 @@ After reading this article, you can:
 To get started, you need to:
 
 1. [Download SQL Server Management Studio (SSMS)](../../ssms/download-sql-server-management-studio-ssms.md). We recommend using a recent version of SSMS with the latest improvements and fixes.
-1. Ensure that your account has the `ALTER ANY EVENT SESSION` [server permission](../../t-sql/statements/grant-server-permissions-transact-sql.md).
+2. Ensure that your account has at least the following permisions: `CREATE ANY EVENT SESSION` or `ALTER ANY EVENT SESSION` [server permission](../../t-sql/statements/grant-server-permissions-transact-sql.md) on versions before SQL Sever 2022
```
Comment (WilliamDAssafMSFT) [inline docs/relational-databases/extended-events/quick-start-extended-events-in-sql-server.md:52]: @AndreasWolter  This is unclear. Do you mean to suggest: In SQL Server 2022, `CREATE ANY EVENT SESSION`. Or before SQL Server 2022, `ALTER ANY EVENT SESSION`?

Comment (WilliamDAssafMSFT) [reply docs/relational-databases/extended-events/quick-start-extended-events-in-sql-server.md:52]: I've made more clear and brought this commit internally.

**Pull request merged**